.travis.yml

```diff
@@ -2,4 +2,20 @@ language: node_js
 node_js:
   - '0.10'
   - '0.11'
+  - '0.12'
+  - '4.0'
+  - "4.1"
+  - "4.2"
+  - "4"
+  - "5.1"
+  - "5.2"
+  - "5.3"
+  - "5.4"
+  - "5.5"
+  - "5.6"
+  - "5.7"
+  - "5.8"
+  - "5.9"
+  - "5"
+  - "6"
```

Comment:
That's a lot of versions. Do we need to run tests on each 'point' release ? Travis CI offers resources on a best effort basis to OpenSource projects. It would be nice to avoid eating too much for a wrapper ;)